workflow/lib/sbs/standardize_barcode_design.py

```diff
@@ -55,16 +68,23 @@ def standardize_barcode_design(
             Case-insensitive matching against gene symbols (default: ["nontargeting", "sg_nt", "non-targeting"])
         nontargeting_format (str): Format string for standardized non-targeting names
             Use {prefix} placeholder for barcode prefix, {original} for original name
+        nontargeting_pattern_map (dict, optional): Mapping of specific patterns to custom formats
+            Example: {
+                "intergenic": "nontargeting_intergenic_{prefix}",
+                "nontargeting": "nontargeting_noncutting_{prefix}"
+            }
+            If None, uses default nontargeting_format for all patterns
         uniprot_data_path (str): Path to UniProt annotation file (REQUIRED for gene validation)
         verbose (bool): Whether to print processing information (default: True)
 
     Returns:
         pd.DataFrame: Standardized barcode design table with columns:
-            - sgRNA: barcode sequences
+            - prefix_map: barcode sequences
             - gene_symbol: gene symbols
             - gene_id: gene identifiers (optional, only if gene_id_col provided)
             - prefix: barcode prefixes for matching
             - uniprot_entry: UniProt entry ID (REQUIRED)
+            - prefix_recomb: recombined barcode sequences (optional, only if prefix_recomb provided)
```

Comment:
```suggestion
            - prefix: barcode prefixes for matching
            - uniprot_entry: UniProt entry ID (REQUIRED)
            - prefix_recomb: recombined barcode sequences (optional)
```

This is obvious, no? or am I misunderstanding the "only if prefix_recomb provided"? or maybe a typo?